### DISCUSSION TITLE
单条日志过大会被截断

### DISCUSSION THREAD

**Discussion by q121102302** [alibaba/ilogtail#756] [Help] [answered]
测试中发现 如果单挑日志过大 会被截断
因为日志有一些其他需求 所以要求尽可能不截断 
请问是否有配置可以调整这个参数？
我使用的版本是1.3.1  
下面是配置
<img width="867" alt="image" src="https://user-images.githubusercontent.com/101638329/229270776-2b415285-be5f-483e-8ff7-68063252276e.png">

**Comment by messixukejia** [comment, marked as answer]
可以参考企业版的参数配置：https://help.aliyun.com/document_detail/64999.html


欢迎将这些解决问题的相关文档提交到社区版

**Reply by q121102302** [reply]
<img width="1070" alt="image" src="https://user-images.githubusercontent.com/101638329/229418897-796c5cd1-9b63-4c5f-b50c-4f5c25a2a722.png">
这个配置 在社区版的ilogtail_config里面增加 是有效的吗？

**Reply by messixukejia** [reply]
可以的

**Reply by q121102302** [reply]
好的 谢谢了
